src/__tests__/integration/basic/nested-write-unsafe-contract.test.ts

```diff
@@ -2,7 +2,7 @@ import fs from 'fs';
 
 import ArLocal from 'arlocal';
 import Arweave from 'arweave';
-import { JWKInterface } from 'arweave/node/lib/wallet';
+import {JWKInterface} from 'arweave/node/lib/wallet';
```

Comment:
wygląda na losową zmianę, zwłaszcza, że zmieniasz formatowanie tylko w kilku miejscach i to na niezgodne z domyślnymi ustawieniami vscode.